### DISCUSSION TITLE
Desktop app bundling

### DISCUSSION THREAD

**Discussion by jacksongoode** [rekkyrosso/ampcast#23] [Ideas]
Just curious if you've considered [tauri](https://tauri.app/) to produce the standalone apps? This would reduce the size of the app itself as well as few other benefits.

**Comment by rekkyrosso** [comment]
The trickiest thing with bundling ampcast is DRM. I've got it working for now but I'm dependent on other projects for that. If Tauri completely solves DRM issues then I might be interested. But I'm not too keen to learn Rust just to bundle my app.
